generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -224,5 +226,5 @@ jhipster:
             enabled: false
             # edit spring.metrics.export.delay-millis to set report frequency
     <%_ } _%>
-    swagger: # swagger is enabled. It can be disabled by pasing 'no-swagger' profile at run time as well
+    swagger: # swagger is enabled by default in dev
         enabled: true
```

Comment:
if this can be controlled purely by profiles then no point having this we can remove it